config/locales/client.en.yml

```diff
@@ -1146,11 +1146,7 @@ en:
         Look for the %{icon} to decide which specific topics, categories and tags you want to be notified about. For more, see your <a href='%{preferencesUrl}'>notification preferences</a>.
       first_notification: "Your first notification! Select it to begin."
       dynamic_favicon: "Show counts on browser icon"
-      skip_new_user_tips:
```

Comment:
Since this is a default off feature, for the time being we need to keep the new user tips overlay as is, right? Since `SiteSetting.enable_onboarding_popups` is `false` by default, we would be disabling the new onboarding tips with this PR. Not sure that is the intention. 

On further thought, it might be good to introduce this in steps: first add the new default off tips just for the timeline and then once the API for that is a bit more solid, add the alternative to the new user tips.